src/metabase/driver.clj

```diff
@@ -411,6 +426,7 @@
   :hierarchy #'hierarchy)
 
 (defmethod supports? :default [_ _] false)
+(defmethod database-supports? :default [driver feature _] (supports? driver feature))
```

Comment:
Style nitpick but it's a little weird interleaving the `:default` implementation of `database-supports?` with all the default implementations of `supports?`.

I would organize this stuff like

```clj
(defmulti supports? ...)

(defmethod supports :default ...)
...

(defmulti database-supports? ...)

(defmethod database-supports? :default 
  [driver feature _] 
  (supports? driver feature))
```

(That's how the rest of the methods are organized in this namespace. Method definition, then default implementation(s). Rinse and repeat)